CHANGELOG.md

```diff
@@ -1,6 +1,92 @@
 # CHANGELOG
 
 -----
+## `v10.0.0-beta`
+### ARM
+In addition to the tabulated changes below, each package had the following updates:
+- Long running operations now run inside a goroutine and return channels for the response and the errors.
+- Some functions changed from returning `autorest.Response` to return the already unmarshaled struct.
+- Uses go-autorest v8.0.0.
+
+| api                                 | version            | note                                |
+|:------------------------------------|:-------------------|:------------------------------------|
+| arm/advisor                         | 2017-04-19         | new                                 |
+| arm/analysisservices                | 2016-05-16         | refactor                            |
+| arm/apimanagement                   | 2016-10-10         | update to latest swagger & refactor |
+| arm/appinsights                     | 2015-05-01         | new                                 |
+| arm/automation                      | 2015-10-31         | new                                 |
+| arm/billing                         | 2017-04-24-preview | update to latest swagger & refactor |
+| arm/cdn                             | 2016-10-02         | refactor                            |
+| arm/commerce                        | 2015-06-01-preview | refactor                            |
+| arm/compute                         | 2016-04-30-preview | refactor                            |
+| arm/consumption                     | 2017-04-24-preview | new                                 |
+| arm/containerregistry               | 2017-06-01-preview | update to latest swagger & refactor |
```

Comment:
This api-version is still under private preview and the SDK is not supposed to be published yet. Can we keep the 2017-03-01 version? /cc @SajayAntony